### ISSUE TITLE
Autism Glossary 

### ISSUE THREAD

**Issue by skfantoni** [alan-turing-institute/AutisticaCitizenScience#542]
## Summary

Deciding how to create the _Information Page_ glossary.

## Date Due by

Next Meet-Up Session 

## Details

I think this is one of the next things we should complete. We need to present it to the group and then we can use it as guidance for deciding what terminology is and isn’t acceptable in regards to the CoC.

## Tasks and Decisions

* [ ] Where do we want to a accumulate information from?
>* [ ] Academic Articles/Journals 
>* [ ] Medical Dictionary 
>* [ ] Charity information pages (ie. NAS)
>* [ ] Our own data 
* [ ] Do we want to include Pros _and_ Cons?
> This would be to specifically explain why some people may find certain terms offensive or upsetting (ie. Asperger, ASD, high-functioning etc.)
* [ ] How do we make it easily understandable to all?
>* [ ] Should we include a Layman version?
>* [ ] Would imaging help?
* [ ] Are we going to include terms that are no longer used?
* [ ] Do we want to write a **Red List** Glossary too

## Who can help

Anyone who wants to :)

**Comment by GeorgiaHCA** [comment]
This is a great issue @skfantoni !
 I found a couple of existing guides which might be a good place to start: 
[NAS](https://www.autism.org.uk/what-we-do/help-and-support/how-to-talk-about-autism)
[Survey (Kenny, Hattersley, Molins, Buckley, Povey & Pellicano, 2016)](https://us.sagepub.com/sites/default/files/autism_terminology_guidelines.pdf)

**Comment by GeorgiaHCA** [comment]
I agree using our own data is good as it reflects the community. There's been a lot of discussion about terminology from previous focus groups which could be useful: https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/master/community-recommendations/focus-groups

**Comment by lottycoupat** [comment]
Great issue!! 👍 @skfantoni

**Comment by GeorgiaHCA** [comment]
This also looks helpful: https://journals.sagepub.com/doi/abs/10.1177/1362361315588200?journalCode=auta

**Comment by skfantoni** [comment]
Thanks @GeorgiaHCA, I’ll have a look over the next few days and see what I can put together

**Comment by skfantoni** [comment]
I've started compiling a google doc based on the links you sent @GeorgiaHCA. So far just a literary review but definitely a step in the right direction you view it [here](https://docs.google.com/document/d/1zZnIbmqfUeufOQRLMV2AvOh3ZSxo-QpTApgw8fM__9c/edit?usp=sharing)

**Comment by GeorgiaHCA** [comment]
Brilliant, thanks so much @skfantoni ! I reckon this would be a good thing to present at a meet-up session?

**Comment by GeorgiaHCA** [comment]
Comment from Sue: please add "neurological" and co-morbidiity" to the glossary. Also consider introducing new term, "placeism"
